packages/react-ui/package.json

```diff
@@ -72,7 +72,8 @@
     "webpack-cli": "^4.5.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.13.9"
+    "@babel/runtime": "^7.13.9",
+    "use-animate-number": "^1.0.5"
```

Comment:
Hi @juandjara is useAnimateNumber a must? Encapsulating animations in a lib might be a good idea, but I've checked it and it has **GPL-3.0-or-later** license https://www.npmjs.com/package/use-animate-number, so I assume it's not compatible with our current carto-react BSD model. Is there any alternative?

Also, not sure you're fully aware of it, but we already have our own `animateValue` function in utils/animations, been used by the current widgets. 

So my point here is that if we finally make a change, I would say we should do first a refactor PR, adding the lib and applying it in the current widgets that already have animated numbers. Otherwise we would have some widgets with a new lib and some others with custom.

Reply:
@juandjara I would remove this new way to use the animated number and discuss and open a new PR (if proceed). Take in mind that a PR implies only one thing, and how we've seen this PR includes more than one.

Reply:
Agree. I'm not against it, but I think it deserves another separated conversation (like the license). Without it, this will be faster

Reply:
@VictorVelarde @efernandezleon as per your comments, I got the concepts separated and created a `useAnimatedNumber` hook based on the `animateValue` util. 
I have pushed the minimal amount of changes to be reviewed in PR #509

Reply:
We'll land first that #509, no prob